functions/src/compute-statistics.ts

```diff
@@ -24,25 +30,38 @@ export const computeStatistics = functions.https.onRequest(
       return;
     }
 
-    await firebaseTools.firestore.delete("stats", {
-      project: config.service_account.project_id,
-      recursive: true,
-      yes: true
-    });
-
     const votes = await db.collection("vote").get();
-    for (const vote of votes.docs) {
-      const voteData = vote.data() as Vote;
-      await updateStats(
-        voteData.value,
-        vote.id,
-        firebase
-          .firestore()
-          .collection(`stats-campaign`)
-          .doc(vote.id),
-        { agency: voteData.agency }
-      );
-    }
-    res.send("done");
+
+    votes.docs
+      .map(vote => {
+        const voteData = vote.data() as Vote;
```

Comment:
Consistent naming with the rest of the app

```suggestion
        const vote = voteSnapshot.data() as Vote;
```